add resource_template capability (#26827)
home-assistant/home-assistant.github.io: source/_integrations/switch.rest.markdown

{% endraw %}

`body_on` and `body_off` can also depend on the state of the system. For example, to enable a remote temperature sensor tracking on a radio thermostat, one has to send the current value of the remote temperature sensor. This can be achieved by using the template `{% raw %}'{"rem_temp":{{states('sensor.bedroom_temp')}}}'{% endraw %}`.

### Switch with a templated resource :

> This example shows a switch that uses a resource [template](/docs/configuration/templating/) to allow Home Assistant to determine its endpoint.

Reviewer: ```suggestion
The following is an example of a switch using a resource [template](/docs/configuration/templating/) to determine the endpoint that Home Assistant should use. 
```
Volunteer comment: Possibly improved word flow.

> It connects to a 3D printer using "fluid" gui, with a (almost) "rest" swith setup, named "LEDs", to switch power on lights. But endpoint varies based on the current light's state.

Reviewer: ```suggestion
It connects to a 3D printer in order to turn its lights on or off. The device uses a REST-like interface (named "LEDs"), but the endpoint varies based on the current state of the light.
```
at a minimum, 'swith' is mispelled, but also trying to improve ease of reading here too.
